src/gf.c

```diff
@@ -3416,7 +3416,7 @@ static int sort_mlmatches(jl_array_t *t, size_t idx, arraylist_t *visited, array
                 addambig = 1; // record there is a least one previously-undetected ambiguity that may need to be investigated later (between m and m2)
             }
         }
-        if (lim == -1 ? msp : !msp2) // include only strong or also weak edges, depending on whether the result size is limited
+        if (include_ambiguous ? msp : !msp2) // include only strong or also weak edges, depending on whether the result size is limited
```

Comment:
```suggestion
        if (0 ? msp : !msp2) // include only strong or also weak edges, depending on whether the result size is limited
```

Reply:
I needed this for a test in test/ambiguous to pass.

Reply:
In CI, it looks like the test/ambiguous tests are failing because of this

Reply:
Oh that's a different test...

Reply:
This seems a very odd condition to choose to branch on the `include_ambiguous` setting though, but some of the tests in ambiguous might be overly picky. I am more worried in this change of the performance implications of doing the more precise sorting. E.g. `@btime methods(Core.kwcall)`.